nextflow.config

```diff
@@ -89,6 +89,8 @@ params {
     max_memory                       = '1.TB'
     max_cpus                         = 32
     max_time                         = '168.h' // 7 days
+    max_spades_retries               = 5
```

Comment:
I would set this to 3, 5 could cause the whole job to take more than week in some cases (which slurm won't like)